docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,157 @@
+# Introduction
+
+In this guide, we'll walk you through the process of setting up your first alert in just a few minutes. You'll witness your alert in action with real-time data, as well as receiving alert notifications.
```

Comment:
It'd be great to structure this section as per https://grafana.com/tutorials/create-alerts-with-logs/